config-model/src/main/java/com/yahoo/vespa/model/builder/xml/dom/DomConfigPayloadBuilder.java

```diff
@@ -249,7 +248,7 @@ private void parseElement(Element currElem, ConfigPayloadBuilder payloadBuilder,
 
     private String verifyLegalOperation(Element currElem) {
         String operation = currElem.getAttribute("operation");
-        if (! Arrays.asList("append", "clear").contains(operation))
+        if (! Arrays.asList("append", "clear").contains(operation.toLowerCase()))
```

Comment:
Just saw that this is used in another place, but it only allows for `append`, so this method should be reverted to original?